### PR TITLE
CSHARP-2684: Server latest removed the “ns” field from index definitions. Fixed tests.

### PR DESCRIPTION
**Pull request by DmitryLukyanov**
Evergreen: https://evergreen.mongodb.com/version/5d51ea2ac9ec4411f82b6061
Evergreen: https://evergreen.mongodb.com/version/5d51ec4f57e85a499e2d6438 (without feature functionality)

### CODE REVIEW

File: src/MongoDB.Driver.Core/Core/Misc/Feature.cs [hunk 1]
```diff
@@ -234,6 +235,11 @@ public class Feature
         /// </summary>
         public static Feature GeoNearCommand => __geoNearCommand;
 
+        /// <summary>
+        /// Gets the getIndexesContainsNS feature.
+        /// </summary>
+        public static Feature GetIndexesContainsNS => __getIndexesContainsNS;
+
```
Comment (rstam) [inline src/MongoDB.Driver.Core/Core/Misc/Feature.cs]: I don't think we should add a Feature for this.

This is only affecting the Legacy API tests and that is not enough of a reason to add a new public Feature to this class.

**Review by rstam — CHANGES_REQUESTED** [review]

File: tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs [hunk 2]
```diff
@@ -663,6 +663,20 @@ public void TestCreateCollectionSetCappedSetMaxSize()
         [Fact]
         public void TestCreateIndex()
         {
+            void AssertNamespace(IndexInfo indexInfo)
```
Comment (rstam) [inline tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs]: I didn't realize on first review that this was a local function.

Therefore AssertNamespace is actually a variable.

Should probably be lowercase.

Feel free to comment if you disagree. I won't hold up approval based on this comment alone.

Comment (DmitryLukyanov) [reply tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs]: Uppercase naming was suggested by resharper rules (not sure which rules are used in our code). But probably you are right. Fixed.

File: src/MongoDB.Driver.Core/Core/Misc/Feature.cs (hunk 1 above)
Comment (DmitryLukyanov) [reply src/MongoDB.Driver.Core/Core/Misc/Feature.cs]: Done

**Review by rstam — APPROVED** [review]
Approving whether AssertNamespace is lower or upper case.

Just need to get this committed to master so I can rebase my commits on this and start getting Evergreen green.

**Pull request closed without merging**